src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -193,15 +196,15 @@ Learn_layout.single_column_layout
         <% papers |> List.iter (fun (paper : Data.Paper.t) -> %>
         <a
           href="<%s paper.links |> List.hd |> (fun x -> x.uri) %>"
-          class="card p-6 flex-1"
+          class="card p-6 flex-1 dark:bg-dark-background_light dark:hover:bg-dark-window_code"
```

Comment:
card background hover color needs to be made a color variable in Figma, then we can use it here. This is definitely not the code window color (or that color is just badly named)